src/components/layout/form/Form.tsx

```diff
@@ -1,9 +1,14 @@
 import React, { FC } from 'react';
+import { FormProps } from 'src/models/forms';
 import { FormStyled } from './FormStyled';
 
-const Form: FC = (props) => {
-  const { children } = props;
-  return <FormStyled component="form">{children}</FormStyled>;
+const Form: FC<FormProps> = (props) => {
+  const { children, onSubmit } = props;
```

Comment:
I believe, you can do like that:
`const Form: FC<FormProps> = ({ children, onSubmit }) => {`